lib/bambora/v1/payment_resource.rb

```diff
@@ -3,15 +3,24 @@
 module Bambora
   module V1
     class PaymentResource
-      # Summary: Create and modify payments.
-      # Note: The link below links to all apis includding profiles and tokenization. There aren't great docs explaining
-      #       the /payments endpoints alone.
-      # Docs: https://dev.na.bambora.com/docs/references/payment_APIs/
-      #       https://dev.na.bambora.com/docs/references/payment_SDKs/take_payments/?shell#
-      # Endpoint: https://api.na.bambora.com/v1/payments
```

Comment:
Moved reference to some of these links to the `README` documentation.

Reply:
Hm, while I think it is important to have it in the `README`, the plus of having this here is with `pry-doc` commands. While navigating the code in a debugger, having these additional doc snippets available would really help.

Reply:
Added this back!